package-managers/qubes-download-dom0-updates.sh

```diff
@@ -129,10 +145,11 @@ if [ ${#PKGLIST[@]} -eq 0 ] && [ "$CHECK_ONLY" = "1" ]; then
 fi
 
 # now, we will download something
-YUM_COMMAND=(fakeroot "${YUM[@]}" "$YUM_ACTION" --downloadonly)
+UPDATE_COMMAND=(fakeroot $UPDATE_CMD $UPDATE_ACTION "${UPDATE_ARGUMENTS[@]}")
+"$UPDATE_CMD" "$UPDATE_ACTION" --help | grep -q downloadonly && UPDATE_COMMAND+=(--downloadonly)
 # check for --downloadonly option - if not supported (Debian), fallback to
 # yumdownloader
-if ! "${YUM[@]}" --help | grep -q downloadonly; then
+if ! "$UPDATE_CMD" --help | grep -q downloadonly && ! "$UPDATE_CMD" --version | grep -q dnf5; then
```

Comment:
While at it, the yumdownload fallback can be dropped - it's relevant for Debian 10, which isn't supported anymore.

Reply:
I deleted the yumdownload block. But will make it a new commit in the same PR. I will also apply shellcheck demands to the same commit.